stock_mts_mto_rule/models/stock_rule.py

```diff
@@ -30,16 +30,19 @@ def _check_mts_mto_rule(self):
                             'rule: %s! It should be the same.') % (rule.name,)
                     raise ValidationError(msg)
 
+    def _get_qty_available_for_mto_qty(self, product, product_uom):
+        src_location_id = self.mts_rule_id.location_src_id.id
+        product_location = product.with_context(location=src_location_id)
+        virtual_available = product_location.virtual_available
+        return product.uom_id._compute_quantity(virtual_available, product_uom)
+
     @api.multi
-    def get_mto_qty_to_order(self, product, product_qty, product_uom, values):
```

Comment:
Maybe you're potentially breaking methods that inherit this one. Is it needed to remove this parameter?

Reply:
Ups, true sorry, I recover this changes